src/client/src/ui/spotTile/spotTileOperations.ts

```diff
@@ -69,7 +68,7 @@ export function spotTileEpicsCreator(executionService$, referenceDataService, op
   }
 
   function onPriceUpdateEpic(action$) {
-    return action$.ofType(PRICING_ACTION_TYPES.PRICING_SERVICE)
+    return action$.ofType(PRICING_ACTION_TYPES.SPOT_PRICES_UPDATE)
       .map((payload) => {
         _.values(payload.payload).forEach((item: SpotPrice) => {
           // TODO: do it better
```

Comment:
This is an old comment, but did you have a chance to see why is it there? It's not a very informative comment.

Reply:
I think that is bad in the first place because is mutating the action data (and all other consumers of that action will see the mutated action if called after). I can take a look and improve it.